components/cookie/Cookie.css

```diff
@@ -0,0 +1,78 @@
+.display-container{
+
+    width: 100%;
+    height: 20%;
+    display: flex;
+    background-color: rgb(255, 255, 255);
+}
+
+
+.cookie-container{
+
+    position: fixed;
+    bottom: 0;
+    text-align: center;
+    border: none;
+    margin-top: 20em;
+    margin-bottom: 0px;
+    height: auto;
+    width: 100%;
+    background-color: #D9D9D9;
+
+}
+
+.description{
+
+    text-align: center;
+    padding: 3px;
+    border: none;
+    color: #1A242D;
+    font-family: 'Mulish';
+    font-size: medium;
+    margin: 10px 10px 10px 10px;
+
+
+}
+
+/* .accept-button{
```

Comment:
Don't commit commented codes.

Reply:
there is small mistake as i'm not save the changes in **cookie.css** before creating the PR now i'm done it